<!DOCTYPE html>
<html>
<head>
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <title>Research</title>
  <link rel="stylesheet" href="styles.css">
  <link rel="icon" type="image/x-icon" href="/images/favicon.ico">
</head>
<body>

  <div class="header">
    <a href="/index.html" class="logo">Joseph Walewski</a>
    <div class="header-right">
      <a href="/about_me.html">About Me</a>
      <a href="/research.html">Research</a>
      <a href="/running.html">Running</a>
      <a href="/news.html">News</a>
    </div>
  </div>

  <div id="first_content">
    <h1>The Ultimate Goal: Applying Newt Regeneration to Humans</h1>
    <p>&nbsp;&nbsp;&nbsp;&nbsp;</p>
  </div>
  <br>  <br>  <br>  <br>  <br>
  <div class="image-container left">
    <img src="/images/yale_medical_stencil_inverted.png" alt="The Entrance to Yale School of Medicine">
    <img src="/images/IMG_1183.jpg" alt="Image">
    <div class="button-container right-button">
      <button class="expand-button">Postgrad Associate Position at <a href="https://medicine.yale.edu/profile/joseph-walewski/">
        Yale Medical School</a></button>
      <div class="expanded-content">
        <h2>Medicine in Action</h2>
        <div class="scrollable-content">
          <p>
            &nbsp;&nbsp;&nbsp;&nbsp;I am currently a postbaccalaureate researcher in the <a href="https://medicine.yale.edu/lab/hafler/">Hafler lab</a> at Yale Medical school. Dr. Hafler is well known for his research on MS (Multiple Sclerosis), a debilitating autoimmune disease of the central nervous system. Thanks to his work we have a much greater understanding of the risk factors involved with the disease’s onset and much more effective treatments against its progression.
          </p>
          <p>    
            &nbsp;&nbsp;&nbsp;&nbsp;My responsibilities in lab are split across all three of my passions: wetlab research, computational analysis, and clinical investigation. 
            My wetlab responsibilities include work under the fume hood isolating Peripheral Blood Mononucleated Cells (PBMCs; these are T cells, B cells, NK cells, and monocytes) from patients with brain tumors for the <a href="https://www.yalemedicine.org/clinical-trials/ab154-combined-with-ab122-for-recurrent-glioblastoma">TIGIT</a> clinical trial. Our goal with this work is to understand the differences in immune response between patients whose tumors originated in the brain (glioblastoma) versus those whose tumor metastasized from elsewhere (brain metastases). Interestingly, patients whose tumor originated elsewhere currently have, on average, much better responses to current treatments and the reasons why are not fully understood.
          </p>
            &nbsp;&nbsp;&nbsp;&nbsp;Additionally, I assist the lab with computational analysis of TCR and BCR (T/B cell receptors, which are types of immune cells) sequences. This is important as these receptors will change depending on the immune response generated, allowing for a greater understanding of how the immune system is reacting to the conditions presented to it. Otherwise, I also perform <a href="https://en.wikipedia.org/wiki/Single-cell_sequencing">sc-RNAseq</a> (single-cell RNA sequencing), <a href="https://en.wikipedia.org/wiki/Trajectory_inference">pseudotime</a>, and <a href="https://en.wikipedia.org/wiki/RNA_velocity">RNA velocity</a> analysis, a subset of this field devoted to inferring developmental directionality by understanding how transcriptional rates are changing within cells. This allows us to understand gene expression, and, more informally, what the cells are doing at any one time and what they might do soon.
             Very recently, I am also exploring spatial proteomic image analysis, which is an actively developing field. On an as-needed basis I assist other lab members with computational needs, such as troubleshooting their computational pipelines and database management. My work has already led to a co-authorship on an article currently in the process of publication, so the link will be here shortly!
          <p>
            &nbsp;&nbsp;&nbsp;&nbsp;Last but certainly not least, I am involved with consenting patients for clinical studies being conducted by Yale Neurology. I thank every patient who has the generosity to donate research samples, as these contributions are how we advance the fields of neurology and immunology to everyone’s benefit. I truly enjoy patient interaction as it brings medical research to life for me. There’s no greater feeling than making an immediate difference in a patient’s day.
          </p>
        </div>
      </div>
      <div class="collapsed-content">

      </div>
    </div>
  </div>


  <div class="image-container left">
    <img src="/images/aqmhd_poster.jpg" alt="Image">
    <div class="button-container">
      <button class="expand-button">Undergraduate Research With <a href="https://www.conncoll.edu/directories/faculty-profiles/stephen-douglass/">
        Professor Douglass</a> at <a href="https://www.conncoll.edu/">Connecticut College</a></button>
      <div class="expanded-content">
        <h2>The Vision Comes to Life</h2>
        <div class="scrollable-content">
          <p>&nbsp;&nbsp;&nbsp;&nbsp;I graduated from Connecticut College in May 2023 with a double major in <a href="https://www.conncoll.edu/academics/majors-departments-programs/majors-and-minors/biochemistry-cellular-and-molecular-biology/">Biochemistry, Cellular, and Molecular Biology</a> (BCMB, Academic distinction) and <a href="https://www.conncoll.edu/academics/majors-departments-programs/majors-and-minors/computer-science/">Computer Science</a> (Honors, senior research award). I had the incredible opportunity to take many challenging courses such as <a href="https://conncoll.smartcatalogiq.com/en/2020-2021/catalog/courses/chm-chemistry/200/chm-223/">organic chemistry</a> and <a href="https://conncoll.smartcatalogiq.com/en/2020-2021/catalog/courses/com-computer-science/300/com-304/">computer algorithms</a>, affectionately called "orgo" and "algo" respectively.
            However, I believe the most defining experience of the undergraduate career to be my research experience.  My undergraduate research with Professor Douglass focused on the topic of <a href="https://www.sciencedirect.com/topics/agricultural-and-biological-sciences/genome-assembly#:~:text=Genome%20assembly%20refers%20to%20the,from%20which%20the%20DNA%20originated.">genome assembly</a>, which involves recovering small snippets of DNA from an organism and piecing them back together in order to recover the original, full length DNA (the genome). Currently existing technology offers two “read” types (either short and accurate or long and less accurate), creating an interesting tradeoff when attempting to stitch together a long and accurate sequence. Luckily, however, it is possible to combine the two read types through a process known as “hybrid genome assembly.”
          </p>
          <p>
            &nbsp;&nbsp;&nbsp;&nbsp;Professor Douglass and I were primarily focused on assembly of large genomes, which offer unique challenges not only due to their size but also their increased complexity. Repetitive elements create confusion over where to put the adjacent sequences as there becomes ambiguity over how much overlap to ignore (for example, do GCTAGTACGATAGTAGTAG”  and “TAGTAGTAGTAGGGCGATA” overlap by one, two, or all three copies of “TAG”?). Once multiple reads with the same repetitive sequence are encountered, the situation becomes even more daunting.
          </p>
          <p>
            &nbsp;&nbsp;&nbsp;&nbsp;The benefits of looking at these types of genomes, however, make the challenges worth it. Many plants and animals have genomes significantly larger than our own, and one example that fascinates me is salamanders. They have incredible regenerative abilities ranging from regrowing entire limbs to restoration of their eyes, heart, and even the brain itself.  By sequencing their genomes, we may gain a greater understanding of how they achieve these incredible feats. Therefore, the question we asked was: “Given the currently available technologies, what is the most cost effective method to assemble a large genome as completely and accurately as possible?” 
          </p>
          <p>
            &nbsp;&nbsp;&nbsp;&nbsp;I spent my time at Connecticut College investigating this question, which ultimately culminated in a <a href="files/woods_hole_conference_poster.pdf">poster</a> presentation at the <a href="https://www.mbl.edu/sites/default/files/2022-10/AQMHD2022-ProgramBook-100622.pdf">Aquatic Models of Human Disease Conference</a> hosted by the <a href="https://www.mbl.edu/">Marine Biological Laboratory</a> of Woods Hole, Massachusetts. There, I was one of four undergraduates to present my work alongside 26 graduate students to be considered for the “best student poster” award. I am both honored and humbled to say that I won first place and was the only undergraduate to receive an award. 
            As of 2024, we are generating novel data to update our results given the release of new sequencing machines in the past year, which has changed the answer of “what’s the most cost effective method to assemble a large genome?” Therefore, there is no publication currently out, but check back soon!
          </p>
        </div>
      </div>
      <div class="collapsed-content">
      </div>
    </div>
    <img src="/images/newt_stencil_inverted.png" alt="An image a newt regenerating its left hind leg">
  </div>


  

  <div class="image-container left">
    <img src="/images/fruit_fly_inverted.png" alt="A fruit fly under the microscope">
    <img src="https://www.biorxiv.org/content/biorxiv/early/2019/02/16/552513/F2.large.jpg" alt="Image">
    <div class="button-container right-button">
      <button class="expand-button"> <a href="https://www.stan-x.org/about-stan-x">STANEX</a> Collaboration at <a href="https://www.exeter.edu/">Phillips Exeter Academy</a></button>
      <div class="expanded-content">
        <h2>My First Research Experience</h2>
        <div class="scrollable-content">
          <p>	
            &nbsp;&nbsp;&nbsp;&nbsp;My first research experience was the final portion of a year long genetics course at Exeter. The fall term assumed no prior genetics knowledge and provided an introduction to heritability, mendelian genetics, and concepts including dominance, recessiveness, incomplete dominance, and codominance. With this information we solved complex heritability problems, learned about the Hardy-Weinberg equilibrium, and began to explore the molecular basis for the phenomena we observed. 
            Throughout the winter term we went into great detail about many of the classic mechanisms of gene regulation throughout all living organisms, ranging from Rho factor in bacteria to chromatin compaction in eukaryotes (more complex organisms such as people, plants, and anything else multicellular). 
          </p>
          <p>
            &nbsp;&nbsp;&nbsp;&nbsp;The spring section of the course was the research experience created by Stanford University. To be able to take this section one had to apply to the course specifically, featuring a 25% acceptance rate amongst students from Exeter dependent on the student’s grades, recommendations from previous biology and chemistry teachers, and a personal statement. I was unbelievably grateful to be admitted to the course and begin my career as a scientist.
          </p>
          <p>
            &nbsp;&nbsp;&nbsp;&nbsp;I was first responsible for husbandry of four drosophila melanogaster breeding lines while class time revolved around understanding the genetic modifications they had. Each line had the same transposable element, the P element, but due to its transposable nature it could move around throughout the genome as different generations of flies came and went in each line. After a few generations, we were told to be on the lookout for “hallelujah males” - male flies which had the transposable element stabilized due to their configuration of homozygous lethal chromosomes.  
            With our lines stabilized to halt further P element movement, we could then characterize the location by sequencing the element and the surrounding portion of the genome. However, since the P element could be nearly anywhere in the genome, we had to be innovative: inverse PCR allowed us to “snip” out the P element and a small portion of the surrounding region, which we could then sequence. We were taught this technique along with fly larva dissection, immunohistochemistry, and immunofluorescence to further study our P element’s expression (usage). 
            Since the P element contained GFP (Green Fluorescent Protein) we could see where the gene was actively being read simply by looking at which parts of the larva lit up. One of my lines had the P element insert next to a gene regulating proton pumps in neurons. The dissection of its brain is the background image for this experience. 
            I am incredibly proud to say that this work resulted in me being listed as a co-author on the paper, which can be found <a href="https://academic.oup.com/g3journal/article/9/7/2097/6026785">here</a> and cited as follows:
          </p>
          <p>
            Lutz Kockel, Griffin C, Ahmed Y, et al. An Interscholastic Network To Generate LexA Enhancer Trap Lines inDrosophila. G3: Genes, Genomes, Genetics. 2019;9(7):2097-2106. doi:https://doi.org/10.1534/g3.119.400105
          </p>
        </div>
      </div>
      <div class="collapsed-content">

      </div>
    </div>
  </div>



  <div class="image-container left">
    <img src="images/newt_cupcake.JPG" alt="A picture of me from June 2010 holding a ceramic cupcake with a newt on top that I designed for art class">
    <div class="button-container">
      <button class="expand-button">Early Life</button>
      <div class="expanded-content">
        <h2>The Start of it All</h2>
        <div class="scrollable-content">
          <p>&nbsp;&nbsp;&nbsp;&nbsp; Even before formally beginning a scientific career I’ve always had both a great passion for and curiosity about the natural world. From the time I could talk I wanted to learn more about the ocean and all life within it, and soon after I discovered some brightly colored salamanders at a family friend’s lake house in upstate New York. After this, I completely fell in love with eastern newts, catching some and watching them navigate both terrestrial and aquatic environments. In the above image, I sculpted a newt onto a ceramic cupcake for a 3rd grade art project.</p>
          <p>&nbsp;&nbsp;&nbsp;&nbsp; Meanwhile, the stencil to the right is from a summer day in 2011 when I was at a local pond with my dad and managed to catch a green frog that, interestingly enough, had five feet. The extra appendage sprouted off the lower right leg, effectively functioning as a second copy. My natural curiosity skyrocketed and I rushed over to my dad, wondering how this could have happened. He told me that it was likely due to chemicals in the environment, but also reminded me that newts, unlike this frog, are capable of regrowing limbs. Therefore, newts might be more resistant to extra limbs appearing due to their greater developmental control. He then handed me some science articles about how amphibians grow limbs and I fell in love with the topic immediately. To this day my primary interest in science is understanding how exactly salamander tissue regeneration happens. One day I hope it would be possible to apply this knowledge to treatment for human conditions ranging from amputee survivors to those affected by neurodegenerative diseases. I see endless possibilities and this is what has motivated me for many years.</p>
        </div>
      </div>
    </div>
      <img src="/images/frog_stencil_test_5.png">
  </div>

  <div class="fixed-footer">
    <div class="container">Built from scratch by Joseph Walewski</div>
  </div>

  <script>
  var collapsibleButtons = document.querySelectorAll('.button-container');
  collapsibleButtons.forEach(function(button) {
      button.addEventListener('click', function () {
          // Toggle the active class for the clicked button
          this.classList.toggle('active');
          
          // Toggle the content visibility for the clicked button
          var content = this.querySelector('.expanded-content'); // Get the content div within the clicked button
          if (content.style.maxHeight) {
              content.style.maxHeight = null;
          } else {
              content.style.maxHeight = content.scrollHeight + 'px';
          }
      });
  });

  </script>
</body>
</html>